engine/build.gradle

```diff
@@ -31,6 +41,9 @@ dependencies {
     compile(group: 'org.terasology.gestalt', name: 'gestalt-entity-system', version: '7.0.5')
     compile(group: 'org.terasology.gestalt', name: 'gestalt-module', version: '7.0.5')
     compile(group: 'org.terasology.gestalt', name: 'gestalt-util', version: '7.0.5')
+    compile group: 'org.terasology.nui', name: 'nui', version: '2.0.0-SNAPSHOT'
+    compile group: 'org.terasology.nui', name: 'nui-libgdx', version: '2.0.0-SNAPSHOT'
+    compile group: 'org.terasology', name: 'TeraMath', version: '1.5.0'
```

Comment:
Did we end up using TeraMath after all?

Reply:
TeraMath shouldn't have been used, so that was probably included in a previous commit (from this pull request) and then left there. Originally the NUI border class would have been in TeraMath but it was moved directly into NUI instead, so TeraMath is not needed. I have now removed the dependency.